R/test.R

```diff
@@ -229,3 +229,59 @@ NULL
 #' @keywords internal
 #' @family tests
 NULL
+
+#' Test case: section highlighting
+#'
+#' @name test-highlight
+#' @keywords internal
+#' @family tests
+#' @details
+#'
+#' R block with R language information.
+#'
+#' ```r
+#' rlang::is_installed()
+#' ```
+#'
+#' YAML block with YAML language information.
+#'
+#' ```yaml
+#' url: https://pkgdown.r-lib.org
+#' footer:
+#'  left:
+#'    structure: [pkgdown]
+#'  right:
+#'    structure: [authors, legal]
+#'    components:
+#' template:
+#'   bootstrap: 4
+#' ```
+#'
+#' R block with no language information.
+#'
+#' ```
+#' rlang::is_installed()
+#' ```
+#'
+#' YAML block with no language information.
+#'
+#' ```
+#' url: https://pkgdown.r-lib.org
+#' footer:
+#'  left:
+#'    structure: [pkgdown]
+#'  right:
+#'    structure: [authors, legal]
+#'    components:
+#' template:
+#'   bootstrap: 4
+#' ```
+#'
+#' YAML block with no language information.
+#'
+#' ```
+#' template:
+#'  params:
+#'    ganalytics: UA-000000-01
+#' ```
```

Comment:
What's the goal of these examples?

Reply:
They are visual tests, should they be removed once there are actual tests?